src/TCommandLine.h

```diff
@@ -93,6 +94,7 @@ public slots:
     bool keybindingMatched(QKeyEvent*);
     void spellCheckWord(QTextCursor& c);
     bool handleCtrlTabChange(QKeyEvent* key, int tabNumber);
+    bool mSpellChecking = false;
```

Comment:
Please put this into the `private` *member variables* (starting a couple of lines down) rather than leaving it in the `private` *methods* list...